packages/react-storage/src/components/StorageBrowser/context/actions/listLocationItemsAction.ts

```diff
@@ -0,0 +1,68 @@
+import { EXTENSIONS } from './constants';
+import {
+  generateString,
+  randomDate,
+  randomNumberInRange,
+  timeout,
+} from './utils';
+
+interface LocationItem {
+  key: string;
+  lastModified: Date;
+  size: number;
+  type: 'FILE' | 'FOLDER';
+}
+
+interface ListLocationItemsActionInput {
+  prefix?: string;
+  options?: {
+    pageSize?: number;
+    nextToken?: string;
+    refresh?: boolean;
+  };
+}
+
+interface ListLocationItemsActionOutput {
+  items: LocationItem[];
+  nextToken: string | undefined;
+}
+
+const generateListLocationItemsData = (count: number): LocationItem[] => {
```

Comment:
just for my understanding, is this function just a way to help you generate location data?

Reply:
Yes, added it for development just for rendering data. Will be used for unit tests in the future